intro.js

```diff
@@ -143,8 +141,8 @@
       //next add intro items without data-step
       //todo: we need a cleanup here, two loops are redundant
       var nextStep = 0;
-      for (var i = 0, elmsLength = allIntroSteps.length; i < elmsLength; i++) {
-        var currentElement = allIntroSteps[i];
+      for (i = 0, elmsLength = allIntroSteps.length; i < elmsLength; i++) {
+        currentElement = allIntroSteps[i];
```

Comment:
Also, we should define `currentElement` within the for-loop, not globally. Use `var` here.